### PR TITLE
chore(docs): update icon docs to new format (VIV-2099)

### PR DESCRIPTION
**Pull request by RichardHelm**

### CODE REVIEW

**Review by TaylorJ76 — CHANGES_REQUESTED** [review]
This UI tests are also failing.

File: libs/components/src/lib/icon/VARIATIONS.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
+## Icon
+
+Vivid comes with a [library of over 1000 icons](/icons/icons-gallery/). You can use these icons in your components by setting the `name` attribute to the icon's name.
+
+```html preview
+<vwc-icon name="heart-solid" aria-hidden="true"></vwc-icon>
+```
+
+### Custom Icon
```
Comment (rachelbt) [inline libs/components/src/lib/icon/VARIATIONS.md:9]: I think that sizes should come bebore this

File: libs/components/src/lib/icon/README.md [hunk 2]
```diff
@@ -1,163 +1,70 @@
...
+| ---------------- | ----------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| **aria-current** | _Enum_:<br/>`page`<br/>`step`<br/>`location`<br/>`date`<br/>`time`<br/>`true`<br/>`false` | Indicates the element that represents the current item within a container or set of related elements.            |
+| **connotation**  | _Enum_:<br/>`accent`<br/>`cta`<br/>`success`<br/>`alert`<br/>`warning`<br/>`information`  | The connotation the icon should have.                                                                            |
+| **size**         | _Enum_:<br/>-6<br/>-5<br/>-4<br/>-3<br/>-2<br/>-1<br/>0<br/>1<br/>2<br/>3<br/>4<br/>5     |                                                                                                                  |
+| **name**         | _Enum_:<br/>`[icon-name]`                                                                 | Indicates which icon to resolve. See the Vivid Icon Gallery for available icons: https://icons.vivid.vonage.com/ |
 
-### Default
+### Events
```
[the remark this answers is not in the document]
Comment (RichardHelm) [reply libs/components/src/lib/icon/README.md:83]: I've copied them from the Vue wrappers but I suppose we can remove the non-relevant ones

File: libs/components/src/lib/icon/VARIATIONS.md (hunk 1 above)
Comment (RichardHelm) [reply libs/components/src/lib/icon/VARIATIONS.md:9]: Well, it's part of the Icon section which should go first I think

**Review by TaylorJ76 — APPROVED** [review]

**Pull request merged**